### ISSUE TITLE
win9x shell and winNT shell behave differently

### ISSUE THREAD

**Issue by bdbaddog** [SCons/scons#586]
This issue was originally created at: 2003-07-21 09:25:37.
This issue was reported by: `shasckaw`.
shasckaw said at 2003-07-21 09:25:37
>In \scon\platform\win32.py, the escape function is
>inadequate to the shell of win98. A command like this
>one fail with "Incorrect command or file name":
>C:\myproject> C:\WINDOWS\command.com "gcc -c -o hello.o
>hello.c"
> 
>To make it work, escape function should be the identity:
>C:\myproject> C:\WINDOWS\command.com gcc -c -o hello.o
>hello.c
> 
>The problem is that on winNT, winXP, win2000, etc, the
>shell is cmd.exe and behave differently than
>command.com, so escape function can't be simply
>replaced: some test must be done before.

issues@scons said at 2003-07-21 09:25:37
>Converted from SourceForge tracker item 775105

mightyllamas said at 2008-04-09 12:04:11
>I don't know where we'd get a machine to test this on, and even if we
>did, I don't know if anyone is still doing development on this platform
>any more.   So - I'm marking this one as wontfix for now.
